src/UserSleep.js

```diff
@@ -0,0 +1,71 @@
+class UserSleep {
+  constructor(userSleepIntake){
+    this.userSleepIntake = userSleepIntake,
+    this.userID = userSleepIntake[0].userID
+  }
+  calculateAverageSleepHours() {
+    let average = this.userSleepIntake.reduce((sum, num) => {
+      sum += num.hoursSlept
+      return sum
+    }, 0)
+    return parseFloat((average / this.userSleepIntake.length).toFixed(2))
+  }
+  calculateAverageSleepQuality() {
+    let average = this.userSleepIntake.reduce((sum, num) => {
+      sum += num.sleepQuality
+      return sum
+    }, 0)
+    return parseFloat((average / this.userSleepIntake.length).toFixed(2))
+  }
+  calculateSleepByDate(date){
+    let sleepForDay =
+    this.userSleepIntake.find(sleepData =>
+    sleepData.date === date)
+    return parseFloat((sleepForDay.hoursSlept).toFixed(2))
```

Comment:
I'd like to take a look at parseFloat and .toFixed together and familiarize ourselves with exactly how they work. They've come in very handy with this project!